src/test/java/feiner/rijks/RijksServiceTest.java

```diff
@@ -0,0 +1,91 @@
+package feiner.rijks;
+
+import json.*;
+import org.junit.jupiter.api.Test;
+import com.andrewoid.ApiKey;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+public class RijksServiceTest {
+    @Test
+    void pageSearch() {
+        // given
+        ApiKey apiKey = new ApiKey();
+        String keyString = apiKey.get();
+        RijksService service = new RijksServiceFactory().getService();
+        int page = 1;
+        // when
+        ArtObjectCollection artObjectCollection = service.pageSearch(keyString, page).blockingGet();
+        // then
+        assertNotNull(artObjectCollection.artObjects);
+        ArtObjects artObjects = artObjectCollection.artObjects[0];
+        assertNotNull(artObjects);
+        assertNotNull(artObjects.title);
+        assertNotNull(artObjects.longTitle);
+        assertNotNull(artObjects.principalOrFirstMaker);
+        assertNotNull(artObjects.webImage);
+        WebImage webImage = artObjects.webImage;
+        assertNotNull(webImage.url);
+        assertNotEquals(-1, webImage.offsetPercentageX);
```

Comment:
0 not -1

Reply:
Put your replies in the pull request, do not reply to the email

Reply:
When I put in 0, the tests failed.

Reply:
You don't need the offset percentages but you should have width and height